### DISCUSSION TITLE
Clean DB and just keep 30days of data ?

### DISCUSSION THREAD

**Discussion by CorrieM** [sipcapture/homer#503] [Q&A] [answered]
how should my heplify-server file look like? if I want to truncate the tables to just keep 30days of data ?
I currently have no disk space left , and I see in the pgsql/12/data/base/16385/ there is every hour a 820mb file :(

This is my current settings

DBDataTable = "homer_data"
DBConfTable = "homer_config"
DBBulk = 100
DBTimer = 4
DBBuffer = 400000
DBWorker = 8
DBRotate = true
DBPartLog = "2h"
DBPartSip = "1h"
DBPartQos = "6h"
DBDropDays = 7
DBDropDaysCall = 0
DBDropDaysRegister = 0
DBDropDaysDefault = 0
DBDropOnStart = true
DBUsageProtection = true
DBUsageScheme = "percentage"
DBPercentageUsage = "80%"
DBMaxSize = "30MB"
Dedup = false

Please assist me with the correct setting to just keep 30days of data.

**Comment by ghost** [comment, marked as answer]
you change DBDropDays = 7 to DBDropDays = 30  and restasrt heplify server

**Comment by williamnsg** [comment]
where i can find the archive with this settings?
